tests/test_sleep_window_parser.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+import unittest
+
+from greenpithumb import sleep_window_parser
+
+
+class TestSleepWindowParser(unittest.TestCase):
+
+    def test_parser(self):
+        sleep_windows = '1:00-3:15,13:45-16:00,23:00-0:30'
+        parsed_windows = sleep_window_parser.parse(sleep_windows)
+        expected_parsed_windows = [
+            (datetime.time(1, 0), datetime.time(3, 15)),
+            (datetime.time(13, 45), datetime.time(16, 0)),
+            (datetime.time(23, 0), datetime.time(0, 30))
+        ]
+
+        self.assertEqual(parsed_windows, expected_parsed_windows)
```

Comment:
Some other interesting test cases:

* `"banana"` (completely wrong input
* `"1:00-"` (incomplete range
* `"1-3"` (incorrectly formatted times: no colon)
* `"1:052-3:059"` (incorrectly formatted times: too many numbers)
* `"5:00-5:00"` (zero-sized range)
* `"5:00:09.897-5:00:15.394"` (unsupported seconds-level precision)
* `"01:00-01:05"` (leading zeroes in front of the hour)
* `"1:5-1:9"` (missing leading zeroes for minutes)